Mathlib/Tactic/Linarith/Verification.lean

```diff
@@ -110,16 +110,16 @@ def mkLTZeroProof : List (Expr × ℕ) → MetaM Expr
       let (iq, h') ← mkSingleCompZeroOf c h
       let (_, t) ← t.foldlM (fun pr ce ↦ step pr.1 pr.2 ce.1 ce.2) (iq, h')
       return t
-  where
-    /--
-    `step c pf npf coeff` assumes that `pf` is a proof of `t1 R1 0` and `npf` is a proof
-    of `t2 R2 0`. It uses `mkSingleCompZeroOf` to prove `t1 + coeff*t2 R 0`, and returns `R`
-    along with this proof.
-    -/
-    step (c : Ineq) (pf npf : Expr) (coeff : ℕ) : MetaM (Ineq × Expr) := do
-      let (iq, h') ← mkSingleCompZeroOf coeff npf
-      let (nm, niq) := addIneq c iq
-      return (niq, ← mkAppM nm #[pf, h'])
+where
```

Comment:
Is that really standard to have a `where` with zero indentation on the left? Same thing several times throughout the PR.

Reply:
I would certainly say it is *more* standard than indenting it. https://github.com/leanprover-community/batteries/blob/f2fb9809751c4646c68c329b14f7d229a93176fd/Batteries/Data/String/Matcher.lean#L49 is one example in batteries.

Reply:
If `where` fits on the previous line, that is obviously nicer - but for definitions which are longer, my impression is that unindented where is the most common. A regex search for `^where\n` finds 80 hits in mathlib.

Reply:
Let's go with this style for now. It also matches what I've been seeing.